src/ImageSharp.Web/FormatUtilities.cs

```diff
@@ -20,6 +20,7 @@ namespace SixLabors.ImageSharp.Web
     public sealed class FormatUtilities
     {
         private readonly IImageFormat[] imageFormats;
+        private readonly Dictionary<IImageFormat, string[]> dottedFileExtensions = new Dictionary<IImageFormat, string[]>();
```

Comment:
Maybe worth indexing on the `IImageFormat.Name.ToLowerInvariant()` as it will make a better index for perf. No dynamic resolution of `GetHashCode()` and `Equals()`. Valid for all indexes here.

Reply:
Thanks 👍 I had just picked some low hanging fruit. I have changed this 🙂